packages/grid/_modules_/grid/models/colDef/gridSelectOperators.ts

```diff
@@ -0,0 +1,42 @@
+import { GridFilterInputValue } from '../../components/panel/filterPanel/GridFilterInputValue';
+import { GridFilterItem } from '../gridFilterItem';
+import { GridFilterOperator } from '../gridFilterOperator';
+
+export const getGridSelectOperators: () => GridFilterOperator[] = () => [
+  {
+    value: 'is',
+    getApplyFilterFn: (filterItem: GridFilterItem) => {
+      if (!filterItem.columnField || !filterItem.value || !filterItem.operatorValue) {
+        return null;
+      }
+
+      return ({ value }): boolean => {
+        return (
+          filterItem.value?.localeCompare((value && value.toString()) || '', undefined, {
```

Comment:
Why not a simple string comparison, e.g. `filterItem.value === value`?

Reply:
This seems to be the approach for the string operators and they are the same as the select ones.

Reply:
Moved to #1968